bin/p2-exec/main.go

```diff
@@ -80,15 +84,35 @@ func main() {
 		}
 	}
 
-	if *launchableName == "" && *cgroupName != "" {
-		log.Fatalf("Specified cgroup name %q, but no launchable name was specified", *cgroupName)
+	if *podID != "" && *launchableName != "" && *launchableCgroupName != "" {
```

Comment:
I don't understand the check for `*launchableName` and `*launchableCgroupName`. They don't seem to be used in this branch. 

Is their usage what the TODO on line 102 is referring to?

Reply:
I guess so, I can remove them until we implement that feature.